warp10/src/main/sh/warp10-standalone.sh

```diff
@@ -388,15 +388,21 @@ start() {
   getConfigFiles
 
   #
-  # Config file exists ?
+  # Config file exists?
   #
   if [[ -z "${CONFIG_FILES}" ]]; then
     echo "Config file does not exist - Use 'bootstrap' command before the very first launch (it must be run as root)"
     echo "WARNING: Since version 2.1.0, Warp 10 can use multiple configuration files. The files have to be present in ${WARP10_CONFIG_DIR}"
     exit 1
   fi
 
-  LEVELDB_HOME="`${JAVACMD} -Xms64m -Xmx64m -XX:+UseG1GC -cp ${WARP10_CP} io.warp10.WarpConfig ${CONFIG_FILES} 'leveldb.home' | grep 'leveldb.home' | sed -e 's/^.*=//'`"
+  //
+  // Extract configuration keys
+  //
+
+  CONFIG_KEYS=$(${JAVACMD} -Xms64m -Xmx64m -XX:+UseG1GC -cp ${WARP10_CP} io.warp10.WarpConfig ${CONFIG_FILES} . 'leveldb.home' 'standalone.host' 'standalone.port'|grep -e '^@CONF@ '|sed -e 's/^@CONF@ //')
```

Comment:
I found it clearer to put a space before and after `|`, all following lines are concerned.